src/app/components/PaymentSummary.tsx

```diff
@@ -11,7 +11,7 @@ function PaymentSummary({ amount, amountAlt, description }: Props) {
         <dt className="uppercase font-semibold text-gray-500 text-xs">
           Amount
         </dt>
-        <dd className="mb-0 dark:text-white">{amount}</dd>
```

Comment:
we should check that globally and it should be sats I think

Reply:
What do you mean we should check that globally? And we decided to use the word 'sat' here: https://github.com/getAlby/lightning-browser-extension/issues/308

Reply:
Ok this will have to be updated everywhere if we are changing from sat to sats, should probably be a separate PR for this?

Reply:
Ok I could do one big PR with changing everything but doing it bit by bit works too.